src/commands.rs

```diff
@@ -366,7 +366,7 @@ struct DescribeArgs {
 
 /// Update the description and create a new change on top.
 #[derive(clap::Args, Clone, Debug)]
-#[command(hide = true)]
+#[command(hide = true, visible_aliases=&["ci"])]
```

Comment:
I was wondering for a second what it means for the command to be hidden but the alias to be visible. I suppose it means that the alias will show up when you do `jj help commit`. Sounds good.